package.json

```diff
@@ -5,7 +5,7 @@
   "repository": "git@github.com:ministryofjustice/use-of-force.git",
   "license": "MIT",
   "scripts": {
-    "copy-views": "cp -R server/views/ dist/server/views/",
```

Comment:
This fixes an intelliJ issue and works in visual studio

Reply:
Just out of interest, what is the issue it fixes?